data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "station"
+	on offer
+		conversation
+			`As soon as you step off your ship a rather imposing Quarg blocks your exit ramp while stairing piercingly at your ship.`
+			`"It is as though a baby in swaddling has killed a dragon, and taken its treasure for its own..." it says with a mix of amusement and disgusted.`
```

Comment:
```suggestion
			`"It is as though a baby in swaddling has killed a dragon, and now rides its corpse, thinking to have mastered fire..." it says with a mix of amusement and disgust.`
```
A more fitting analogy, imo, for both the situation and the ship. You haven't looted something the dragon guards, you have desecrated and weapon used the dragon's own body.

Reply:
Would changing "killed a dragon" to "tamed a dragon" work too?

So 

"It is as though a baby in swaddling has tamed a dragon, and now rides it through the sky spewing fire with reckless abandon."